lib/metrics/StatsClient.js

```diff
@@ -65,15 +65,30 @@ class StatsClient {
     /**
     * report/record a new request received on the server
     * @param {string} id - service identifier
-    * @param {callback} cb - callback
+    * @param {number} incr - optional param increment
+    * @param {function} cb - callback
     * @return {undefined}
     */
-    reportNewRequest(id, cb) {
+    reportNewRequest(id, incr, cb) {
         if (!this._redis) {
             return undefined;
         }
-        const callback = cb || this._noop;
+
+        let callback = this._noop;
```

Comment:
may just declare `let callback` here and at line 83 do `callback = ... : this._noop;`

Same thing for `amount` if you add `amount = 1;` in the `typeof incr === 'function'` block (I think it's clearer not to reassign to the same variable previously assigned so one does not have to backtrack to the original value).